pkg/admin/model/resp/resp.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package resp
+
+import (
+	"github.com/apache/dubbo-kubernetes/pkg/admin/model/req"
+	"github.com/vcraescu/go-paginator"
+)
+
+type BaseResp struct {
+	Code    Code        `json:"code"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data"`
+}
+
+type SuccessResp struct {
+	BaseResp
+}
+
+type ErrorResp struct {
+	BaseResp
+}
+
+type Code int
+
+// TODO: refactor this
+const (
```

Comment:
For now, I think the error code can be divided into three level: normal, warn, critical. For example:  200 ~ 210 can be classified to normal, 211 ~ 220 can be classified to warn, 221~230 can be classified to critical, so that the errors can be shown in the front by different colors to tell user the result of important operations.